elementary/clients/dbt/dbt_log.py

```diff
@@ -0,0 +1,38 @@
+
+from dataclasses import dataclass
+import json
+from typing import Iterator, Optional
+
+from elementary.utils.log import get_logger
+
+logger = get_logger(__name__)
+
+
+@dataclass
+class DbtLog:
+    msg: Optional[str]
+    level: Optional[str]
+    exception: Optional[str]
+
+    @classmethod
+    def from_log_line(cls, log_line: str) -> 'DbtLog':
+        log = json.loads(log_line)
+        return cls(
+            msg=log.get("info", {}).get("msg") or log.get("data", {}).get("msg"),
+            level=log.get("info", {}).get("level") or log.get("level"),
+            exception=log.get("info", {}).get("exc") or log.get("data", {}).get("exc")
+        )
+    
+    def __str__(self) -> str:
+        ret = f"{self.level or 'unknown'}: {self.msg}"
+        if self.exception:
+            ret += f"\nException:\n{self.exception}"
```

Comment:
I think this should say `Error:` as it's clearer to users than `Exception` which is a coding-term.

Reply:
On it